examples/generic-azure.yaml

```diff
@@ -673,3 +673,39 @@ groups:
           ## References
           - section 9.1.2 https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/
         scored: true
+      - id: 1.5.3
+        description: "<sid>adm user and its <sid> group exist and enabled"
+        audit: |
+          # Get SIDs from cibadmin
+          IFS=' ' read -r -a sid_list_raw <<< $(sudo cibadmin -Q | xmllint -xpath '//primitive//nvpair[@name="SID"]/@value' -)
```

Comment:
Looks good for me! I like that change. It addresses both the multi-sid and the nsswitch problematic.
Maybe we need to get more allowed values in nsswitch.conf but for now I would say we are fine.

Reply:
Thank you!)